.github/workflows/rust-test.yml

```diff
@@ -74,6 +69,7 @@ jobs:
             ${{ matrix.build }}-${{ env.cache-name }}-${{ steps.toolchain.outputs.rustc_hash }}-rust-${{ hashFiles('**/Cargo.lock') }}
             ${{ matrix.build }}-${{ env.cache-name }}-${{ steps.toolchain.outputs.rustc_hash }}-
             ${{ matrix.build }}-${{ env.cache-name }}-
+            ${{ matrix.build }}-
```

Comment:
@glasser I don't understand why we would ever fallback to a cache that isn't seeded with our cache-buster value, but happy to continue this after this lands.

Reply:
This was copied from the slow test file.

Reply:
Gotcha. Thanks!

I had tried to remove these having encountered them previously and questioning them and thought you were re-introducing on account of that but this sounds more like the very real perils of the multiple nearly identical but subtly varying confits we had before.  So I'm glad we're whittling those down!  Might still propose removing those but I've also employed a bigger hammer in https://github.com/apollographql/federation/pull/649, should it become necessary in the short term.